third_party/3/pyspark/sql/dataframe.pyi

```diff
@@ -91,17 +92,17 @@ class DataFrame:
     def selectExpr(self, *expr: List[str]) -> 'DataFrame': ...
     def filter(self, condition: ColumnOrName) -> 'DataFrame': ...
     @overload
-    def groupBy(self, *cols: ColumnOrName) -> 'DataFrame': ...
+    def groupBy(self, *cols: ColumnOrName) -> GroupedData': ...
```

Comment:
There is a dangling quote in the return type annotation here.

Reply:
Yeah, I did it in a text editor as I didn't have a proper IDE. Thanks for taking a look, it was on my backlog!